src/views/Settings.vue

```diff
@@ -92,13 +92,60 @@
           >{{ isChangingPassword ? 'Changing password...' : 'Change password'}}</b-button>
         </card-widget>
       </b-col>
+      <b-col col cols="12" md="6" xl="4">
+        <card-widget header="System">
+          <div class="pt-2">
+            <div class="d-flex w-100 justify-content-between px-3 px-lg-4 mb-4">
+              <div>
+                <span class="d-block">Shutdown</span>
+                <small class="d-block" style="opacity: 0.4">Power off your Umbrel</small>
```

Comment:
Tip: You can use class `text-muted` for a nicely faded text. It's how the rest of faded text is in dashboard :)

Reply:
That's just direct copypasta from the Tor markup haha
https://github.com/getumbrel/umbrel-dashboard/blob/6506757ca7395b9e0394338f13ade6c81b505714/src/views/Settings.vue#L20

Reply:
lolol, no idea why I did that 🤦‍♂️